src/pages/instantlaunch/[id]/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @author mian
+ *
+ * Instant launch launch page
+ */
+import React from "react";
+
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+import { i18n, RequiredNamespaces } from "i18n";
+
+import { useRouter } from "next/router";
+
+import InstantLaunchStandalone from "components/instantlaunches/launch";
+
+export default function InstantLaunch() {
+    const router = useRouter();
+    const { id } = router.query;
+
+    return <InstantLaunchStandalone id={id} />;
+}
+
+export async function getServerSideProps({ locale }) {
+    const title = i18n?.t("instantLaunches") || "";
```

Comment:
As mentioned in slack, I'm not sure if this is an issue outside of dev mode (though it could still be an issue for the very first user to hit a non-root-page when the nextjs server first starts up), but I think the fall back string for this title can be the English string `"Instant Launches"` in this case, rather than `""`.

Reply:
Yeah, that's a good point. I realize I should probably add a string for this page title specifically anyway, since the plural "Instant Launches" doesn't make that much sense for a single-instant-launch page anyway. I think you're right that this is basically only a dev-mode problem so I think I'll just remove the `?.` and fallback here to avoid duplicating the string.